README.rst

```diff
@@ -96,14 +96,15 @@ Quick Example
     from nextcord.ext import commands
 
 
-    bot = commands.Bot(command_prefix='$')
+    bot = commands.Bot()
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
-
-    bot.run('token')
 
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
+        
+    bot.run("token")
+    
```

Comment:
Confused as to why there is trailing whitespace now

Reply:
Is it better now?

Reply:
you just removed another line, add that back
![image](https://user-images.githubusercontent.com/34910574/180519984-bdbc0ae0-e8d3-4da6-9a7e-53f250a5f240.png)